packages/plaid-funding-source/pages/index.tsx

```diff
@@ -164,7 +203,14 @@ const Home: NextPage = () => {
                                     placeholder="Enter your Dwolla funding source name"
                                 />
                             </Form.Group>
-
+                            <Form.Group as={Row} className="mb-3" controlId="formGroupFundingSourceName">
```

Comment:
Can we change the `controlId` to something like `formGroupOnDemandChecked` since the current value is already used?

Reply:
Good catch!